recipes-debian/vte/vte_debian.bb

```diff
@@ -0,0 +1,65 @@
+# base recipe: meta/recipes-support/vte/vte_0.52.2.bb
+# base branch: warrior
+# base commit: 9516e8117b9548890d0c3e96906d29dbdbed28a8
+
+SUMMARY = "Virtual terminal emulator GTK+ widget library"
+BUGTRACKER = "https://bugzilla.gnome.org/buglist.cgi?product=vte"
+LICENSE = "GPLv3|LGPLv2.1+|LGPLv3"
+DEPENDS = "glib-2.0 gtk+3 libpcre2 intltool-native libxml2-native gperf-native"
+
+LIC_FILES_CHKSUM = "\
+	file://COPYING.GPL3;md5=2f31b266d3440dd7ee50f92cf67d8e6c \
+	file://COPYING.LGPL2;md5=4fbd65380cdd255951079008b364516c \
+	file://COPYING.LGPL3;md5=b52f2d57d10c4f7ee67a7eb9615d5d24 \
+"
+
+# this overwrites SRC_URI, so inherit before debian-package.
+inherit gnomebase
+
+BPN = "vte2.91"
+inherit debian-package
+require recipes-debian/sources/${BPN}.inc
+DEBIAN_UNPACK_DIR = "${WORKDIR}/vte-${PV}"
+
+inherit gtk-doc distro_features_check upstream-version-is-even gobject-introspection
+
+# vapigen.m4 is required when vala is not present (but the one from vala should be used normally)
+FILESPATH_append = ":${COREBASE}/meta/recipes-support/vte/vte"
+SRC_URI += "\
+	file://0001-Don-t-enable-stack-protection-by-default.patch \
+	${@bb.utils.contains('PACKAGECONFIG', 'vala', '', 'file://0001-Add-m4-vapigen.m4.patch', d) } \
+	file://0001-app.cc-use-old-school-asignment-to-avoid-gcc-4.8-err.patch \
+"
+
+ANY_OF_DISTRO_FEATURES = "${GTK3DISTROFEATURES}"
+
+# Instead of "inherit vala" we do the relevant settings here so we can
+# set DEPENDS based on PACKAGECONFIG.
+
+# Our patched version of Vala looks in STAGING_DATADIR for .vapi files
+# export STAGING_DATADIR
```

Comment:
According to the "Our patched version of Val ~"  line, STAGING_DATADIR variable need to be exported for Vala package.,

Reply:
I have changed to set $STAGING_DATADIR.